src/cronParser.ts

```diff
@@ -316,19 +316,18 @@ export class CronParser {
   }
 
   protected validate(parsed: string[]) {
-    const standardCronPartCharacters = "0-9,\\-*\/";
+    const standardCronPartCharacters = "0-9,\\-*/";
     this.validateOnlyExpectedCharactersFound(parsed[0], standardCronPartCharacters);
     this.validateOnlyExpectedCharactersFound(parsed[1], standardCronPartCharacters);
     this.validateOnlyExpectedCharactersFound(parsed[2], standardCronPartCharacters);
     // DOM
-    this.validateOnlyExpectedCharactersFound(parsed[3], "0-9,\\-*\/LW");
+    this.validateOnlyExpectedCharactersFound(parsed[3], "0-9,\\-*/LW");
     this.validateOnlyExpectedCharactersFound(parsed[4], standardCronPartCharacters);
     // DOW
-    this.validateOnlyExpectedCharactersFound(parsed[5], "0-9,\\-*\/L#");
+    this.validateOnlyExpectedCharactersFound(parsed[5], "0-9,\\-*/L#");
```

Comment:
Why are these change needed?

Reply:
I'll double check. I should have paid more attention to changes the linter was suggesting. It may not be necessary.

Reply:
Removed in a22ef7b2bafa467daed175084a7885a1f9e72e58